src/content/es/fundamentals/codelabs/your-first-pwapp/index.md

```diff
@@ -812,11 +818,11 @@ deferredInstallPrompt.prompt();
 evt.srcElement.setAttribute('hidden', true);
 ```
 
-Al llamar a `.prompt()` se mostrará un diálogo modal al usuario y se le pedirá que agregue su aplicación a la pantalla de inicio.
+Al llamar a `.prompt()` se mostrará un diálogo modal al usuario y se le pedirá que agrega tu aplicación a la pantalla de inicio.
 
 ### Registrar los resultados
 
-Usted puede comprobar para ver cómo el usuario responde al cuadro de diálogo de instalación mediante la escucha de la promesa de regresar por el `userChoice` propiedad de los salvados `beforeinstallprompt` evento. La promesa devuelve un objeto con una propiedad `outcome` después de que se haya mostrado la solicitud y el usuario haya respondido a ella.
+Puede verificar qué respondió el usuario al cuadro de diálogo de instalación escuchando la promesa que devuelve la propiedad `userChoice` del evento `beforeinstallprompt` salvado. La promesa devuelve un objeto con una propiedad `outcome` después de que se haya mostrado la solicitud y el usuario haya respondido a ella.
```

Comment:
`Pues verificar...`  to keep the conjugation in the second person.

Reply:
I changed by `Puedes verificar`